app/drivers/kscan/Kconfig

```diff
@@ -54,6 +60,30 @@ config ZMK_KSCAN_MATRIX_WAIT_BETWEEN_OUTPUTS
 
 endif # ZMK_KSCAN_GPIO_MATRIX
 
+if ZMK_KSCAN_GPIO_ROUND_ROBIN
+
+config ZMK_KSCAN_MATRIX_WAIT_BEFORE_INPUTS
```

Comment:
I'm not sure there's much value in duplicating these values, or using shared config entries for these two properties. Thoughts?

Reply:
I thought that initially as well, but since multiple `kscan`s can be used, I thought it best to keep them seperate. I note that this one is a copy paste error though, it should be `CONFIG_ZMK_KSCAN_ROUND_ROBIN_WAIT_BEFORE_INPUTS`.

I'm easy either way. Can fix the typo, or rely on the MATRIX version. Your call.

Reply:
@petejohanson waiting for your opinion on this one. Have done the typo fix for now.